.github/workflows/bundle_with_dakota.yml

```diff
@@ -0,0 +1,161 @@
+name: 🏎️ Make & Test Wheels 🏎️
+
+on: [pull_request]
+
+env:
+  ERT_SHOW_BACKTRACE: 1
+  NO_PROJECT_RES: 1
+  BOOST_VERSION: 1.82.0
+  BOOST_VERSION_UNDERSCORES: 1_82_0
```

Comment:
Boost 1.83 was released some time ago. There really is no reason not to use the newest version.
Bleeding build is already using 1.83.

Reply:
Built & tested with 1.83, works